### ISSUE TITLE
feature request: switch desktops

### ISSUE THREAD

**Issue by lovitus** [TransposonY/GestureSign#115]
dear dev 
# feature
wish to add guesture to switch between desktops   ( task view --> desktops)
# scene:
I normally open multiple full screen app and remote desktops and put in difference virtual desktops , I wish to have this app to switch desktops (like 4 fingers swipe gesture of precise touchpad )

will you consider this feature ?

**Comment by kabylake2** [comment]
This gesture is built-in windows

**Comment by Etrit1** [comment]
Yes, it's built in Windows.  You can ALSO change it by setting up a gesture to do a keyboard shortcut  and just disable the swipe for it in Windows.

Seriously, you can do almost anything with this by using keyboard shortcuts.   The only limit is how many unique gestures you can come up with that DONT end up confusing it. (which is the only limiting factors)

I wish it was like BetterTouchtool for Mac used to be that would pay attention to the order in which you tapped your fingers also.  That gave you the ability to say, have a 3 finger tap all at once mean something different then tap hold with 1 finger then tap with the other two fingers.   That is much easier then drawing with one finger while holding down with two others but whatever, still is pretty great.
